dressForTheWeather/Models/Outfit.swift

```diff
@@ -12,7 +12,7 @@ struct Outfit {
     
     var components: [ClothingItem]
     
-    var recommendations: String {
+    var recommendationString: String {
```

Comment:
If this name gets updated it shouldn't have its type in the name, but what it represents instead. If you go to this site it has the rules for the naming conventions we've been following with the Raywenderlich style guide: https://swift.org/documentation/api-design-guidelines/

There's a brief part titled, "Name variables, parameters, and associated types according to their roles, rather than their type constraints." which talks about this a little.

Reply:
Good call. I think I originally had `recommendations` as an actual array of objects, so naming this something else for clarity seemed like a good idea at the time. 

Updated to `recommendationDescription` in b629c44f9cacf9103d55795c952e422ea855ecde. I think that's accurate to how it's used.